app/views/admin/complete_persons.html.erb

```diff
@@ -1,25 +1,23 @@
 <% provide(:title, 'Finish unfinished persons') %>
 
 <div class="container">
-  <% competition_ids = @finish_persons.competition_ids %>
-
   <h2>
     <%= yield(:title) %>
-    <%= link_to "Go back", admin_finish_unfinished_persons_path(competition_ids: competition_ids), class: "btn btn-default" %>
+    <%= link_to "Go back", panel_page_path(id: User.panel_pages[:createNewComers], competition_ids: @competition_ids.join(',')), class: "btn btn-default" %>
   </h2>
 
-  <% if competition_ids.present? %>
+  <% if @competition_ids.present? %>
     <p>For competition(s):</p>
 
     <ul>
-      <% @finish_persons.competitions.each do |competition_id| %>
+      <% @competition_ids.each do |competition_id| %>
         <li><code><%= competition_id %></code></li>
       <% end %>
     </ul>
   <% end %>
 
   <%= simple_form_for :person_completions, url: admin_complete_persons_path do |f| %>
-    <%= f.hidden_field :competition_ids, value: competition_ids %>
+    <%= f.hidden_field :competition_ids, value: @competition_ids %>
```

Comment:
Again, this creates an `input` field in the DOM tree. How is this being handled if `@competition_ids` is an array?

Reply:
(In other words: Should you manually glue them together with `join(',')` or is that not necessary?

Reply:
Thanks for the catch. I fixed it now.

I tested with one competition and no competition. When I input no competition, it will run this script for all competitions, and I wrongly assumed that multiple competitions case will also be covered there.